src/ctree-app/ctree-landing.html

```diff
@@ -278,30 +407,32 @@
   </template>
 
   <script>
-    class CTreeLanding extends Polymer.mixinBehaviors([Polymer.IronScrollTargetBehavior], Polymer.Element) {
-
-      static get is() { return 'ctree-landing'; }
+    class CTreeLanding extends Polymer.mixinBehaviors(
+      [Polymer.IronScrollTargetBehavior],
+      Polymer.Element
+    ) {
+      static get is() {
+        return "ctree-landing";
+      }
 
       static get properties() {
         return {
           rootPath: {
             type: String,
-            value: '',
+            value: ""
           },
 
           section: String,
 
           _suppressInferSection: {
             type: Boolean,
-            value: false,
-          },
+            value: false
+          }
         };
       }
 
       static get observers() {
-        return [
-          '_dialogParamChanged(queryParams.section)',
-        ];
+        return ["_dialogParamChanged(queryParams.section)"];
```

Comment:
The multi line format is the standard Polymer convention because there may be additional items added.